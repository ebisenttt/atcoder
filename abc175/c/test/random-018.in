747105478822039 241891843274700 249895105197779
