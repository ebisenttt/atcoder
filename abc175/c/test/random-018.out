247315268426481
